sandbox/load_test_jobs.py

```
@@ -15,87 +15,922 @@
+    if "--help" in raw or "-h" in raw:
+        return list(raw)
+    cmd: Optional[str] = None
+    rest: List[str] = []
+    for token in raw:
+        if cmd is None and token in KNOWN_SUBCOMMANDS:
+            cmd = token
+        else:
+            rest.append(token)
+    if cmd is None:
+        cmd = "simple"
+    return [cmd] + rest
```

Ruff (0.15.13)

[warning] Consider `[cmd, *rest]` instead of concatenation

Replace with `[cmd, *rest]`

(RUF005)